src/router.js

```diff
@@ -46,7 +46,7 @@ export class Router {
     }
   }
 
-  configure(callbackOrConfig) {
+  configure(callbackOrConfig:any):Router {
```

Comment:
`callbackOrConfig` is a `RouterConfiguration` or a callback that takes a `RouterConfiguration`.